pennylane_qiskit/ibmq.py

```diff
@@ -82,14 +85,20 @@ def batch_execute(self, circuits):  # pragma: no cover
     def _track_run(self):  # pragma: no cover
         """Provide runtime information."""
 
+        expected_keys = {"created", "running", "finished"}
```

Comment:
Does Qiskit provide those keys somewhere?

Reply:
I dug deep into API docs to find [this](https://cloud.ibm.com/apidocs/quantum-computing#get-job-metrics-jid-response). If you expand the `timestamps` drop-down in the `Response` category, you see these keys. Of course, you'd only check that if you found that `_time_per_step` gets its metadata from the `timestamps` endpoint by digging through source code (ctrl-F for "timestamps" in [this file](https://qiskit.org/ecosystem/ibm-provider/_modules/qiskit_ibm_provider/job/ibm_circuit_job.html#IBMCircuitJob.time_per_step))

Reply:
Ok let's keep as it is then